### ISSUE TITLE
Doxygen fails to compile LaTeX formulas

### ISSUE THREAD

**Issue by dangars** [dealii/dealii#7623]
There are several places in the online documentation with `Undefined control sequence`. I order to find the classes with problems you can google
```
Undefined control sequence site:https://www.dealii.org/9.0.0/doxygen/deal.II
```
I have not been able to identify the source of the problem because there are no errors when I generate the doxygen documentation in my computer.

**Comment by tjhei** [comment]
Thanks for pointing this out! Some of these still exist on master. You can not find "Undefined control sequence" as this is generated by mathjax at runtime.

I think we should be able to see this by switching of mathjax locally. I will try this out...

**Comment by tjhei** [comment]
I fixed some in #7624 but there are a ton more:
```
$ grep Undefined ./doc/doxygen/deal.II/_formulas.log | wc -l
162
```
I will work on them later today...

**Comment by tjhei** [comment]
This is now fixed in #7625. With both PRs, I get 0 broken formula errors now. I think we can close this?

**Comment by dangars** [comment]
I agree we can close it :)
